examples/static/samples/json/examples2.json

```diff
@@ -0,0 +1,20 @@
+{
+ "items": [
+   {
+     "title": "amp-bind",
+     "url": "/components/amp-bind/"
```

Comment:
Do these need to be migrated?

Reply:
Not really, we could also use the resources from abe.com - up to you!

Reply:
What I mean is: won't the URLs be broken in the sample.

Reply:
Fixed with 60ac96e.